docs/examples/getting_started.ipynb

```diff
@@ -419,7 +419,15 @@
     "\n",
     "Rotate's `angle` parameter can accept `float` or `float tensor` types of values. The second option, `float tensor`, enables us to feed the operator with different rotation angles for every image, via a tensor produced by other operation.\n",
     "\n",
-    "Random number generators are examples of operations that one can use with DALI. Let us use `fn.random.uniform` to make a pipeline that rotates images by a random angle."
+    "Random number generators are examples of operations that one can use with DALI. Let us use `fn.random.uniform` to make a pipeline that rotates images by a random angle.\n",
+    "\n",
+    "> **_NOTE_**\n",
+    ">\n",
+    "> Keep in mind, that every time you pass an output of a DALI operator to another as a named keyword argument (like the angle for the rotate above) that data must be placed on the CPU (``random.uniform`` has a default device='cpu').\n",
+    ">\n",
+    "> Only regular inputs (non-named, positional ones) can use either CPU or GPU data, as show below.\n",
+    ">\n",
+    "> More information can be found in the [pipeline documentation section](../pipeline.html?#processing-graph-structure)."
```

Comment:
```suggestion
    "> Keep in mind, that every time you pass an output of a DALI operator to another as a named keyword argument that data must be placed on the CPU. In the example below, we use the output of ``random.uniform`` (which has a default device='cpu') as ``angle`` keyword argument to ``rotate``.\n",
    ">\n",
    "> Such arguments in DALI are called "argument inputs", .more information about them can be found in the [pipeline documentation section](../pipeline.html?#processing-graph-structure)."
    "> Regular inputs (non-named, positional ones) do not have such constraints and can use either CPU or GPU data, as show below.\n",
    ">\n",
```
The part about them being argument inputs is not necessary, but I think the first sentence will be ordered better with what I propose.

Reply:
Fixed